packages/common/src/store/buy-usdc/sagas.ts

```diff
@@ -458,49 +458,19 @@ function* recoverPurchaseIfNecessary() {
       })
     )
 
-    yield* call(
-      retry,
-      async () => {
-        const transferTransaction = await createRootWalletRecoveryTransaction(
-          audiusBackendInstance,
-          {
-            wallet: rootAccount,
-            userBank,
-            amount,
-            feePayer: feePayerKey
-          }
-        )
-        transferTransaction.partialSign(rootAccount)
-
-        console.debug(`Starting root wallet USDC recovery transaction...`)
-        const { res, error } = await relayTransaction(audiusBackendInstance, {
-          transaction: transferTransaction
-        })
+    const user = yield* select(getAccountUser)
+    const ethWallet = user?.wallet
+    if (!ethWallet) {
+      throw new Error('User is not signed in')
```

Comment:
nit: I think we use this exact message in a few places and it's slightly obscuring the actual problem (we didn't find a wallet).

Reply:
Would you suggest `User not found or does not have a wallet?` I'm not sure why wallet is optional on the user object. Is it ever not present? IIRC it's one of the mandatory fields for a user record

Reply:
went with `Unable to find wallet. Is the user signed in?` lmk what you think